test/scripts/models/post.js

```diff
@@ -29,13 +29,13 @@ describe('Post', () => {
     }).then(data => {
       data.title.should.eql('');
       data.date.valueOf().should.gte(now);
-      data.updated.valueOf().should.gte(now);
       data.comments.should.be.true;
       data.layout.should.eql('post');
       data._content.should.eql('');
       data.link.should.eql('');
       data.raw.should.eql('');
       data.published.should.be.true;
+      should.not.exist(data.updated);
```

Comment:
In actual use, `data.updated` won't be null, thanks to https://github.com/hexojs/hexo/blob/2146a03a2cfced62ecfb623e5da99f21269711f6/lib/plugins/processor/post.js#L90.